app/src/main/java/com/github/polypoly/app/base/game/rules_and_lobby/kotlin/GameRules.kt

```diff
@@ -26,12 +24,14 @@ data class GameRules (
     val initialPlayerBalance: Int = 500,
 ) {
 
+    private val maxRoundHours = Int.MAX_VALUE
```

Comment:
It's interesting to restrict the time that can be put in, it's just that it's not useful to initialize it to `Int.MAX_VALUE`, but I guess you just did that while waiting for the group to discuss it. Maybe put a `TODO` so we don't forget to change it!